.prettierignore

```diff
@@ -1,5 +1,8 @@
 lib/
 pnpm-lock.yaml
 
+src/nodes/typeGuards/simple.ts
+src/nodes/typeGuards/union.ts
```

Comment:
Why?

If these should be ignored by Prettier in some way, I'd rather use [Prettier's inline directives](https://prettier.io/docs/en/ignore.html#javascript). But the best case would be to not ignore them.

If the goal here was to keep the functions all on one line, I'm -1 on that. The Prettier-standardized formatting makes the repo much easier for me to read through. 😄 

If the multiple lines per thing really bothers you, I wonder if something like this would be nicer?

```ts
export const isAbstractKeyword = createSimpleTypeGuard(ts.SyntaxKind.AbstractKeyword);
```